app/src/main/java/de/tum/in/tumcampusapp/component/ui/ticket/model/Event.kt

```diff
@@ -34,6 +35,18 @@ data class Event(@PrimaryKey
         const val defaultDuration = 7200000 // Milliseconds
     }
 
+    fun getFormattedDateTime(): String {
+        return "${getFormattedDate()}, ${getFormattedTime()}"
+    }
+
+    fun getFormattedDate(): String {
+        return DateTimeFormat.forPattern("dd MMMM yyyy").print(start)
```

Comment:
You are not respecting locale here. Instead something like `DateTimeFormat.mediumDate` should be used instead of statically setting the format

Reply:
Right, will change that.